db/schema.rb

```diff
@@ -10,22 +10,33 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_09_28_143747) do
+ActiveRecord::Schema.define(version: 2018_09_26_143501) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "books", force: :cascade do |t|
-    t.string "genre", null: false
-    t.string "author", null: false
+    t.string "genre"
```

Comment:
You have some contradiction on you migration, maybe you need a rebase or a merge.